src/test/java/com/hillel/items_exchange/EvoAppApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.hillel.items_exchange;
+
+import com.github.database.rider.spring.api.DBRider;
+import org.junit.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+@DBRider
+@AutoConfigureMockMvc
+public class EvoAppApplicationTests {
```

Comment:
Would you mind to completely remove this class?

Reply:
This class tests wed application context. Now it works correctly, because the annotation to JUnit5